### PR TITLE
ArcGIS API for Python 1.9.0 release notes

### PR DESCRIPTION
**Pull request by jyaistMap**
Here's an initial draft for release notes up to date as of 6.16
* @priyankatuteja Can you please review the text in the `arcgis.learn` module
* @rhea-j Can you review the text for all the changes pertaining to `arcgis.raster` and its submodules

-----

# Checklist

Please go through each entry in the below checklist and mark an 'X' if that condition has been met. Every entry should be marked with an 'X' to be get the Pull Request approved.

- [ ] All `import`s are in the first cell? First block of imports are standard libraries, second block are 3rd party libraries, third block are all `arcgis` imports? Note that in some cases, for samples, it is a good idea to keep the imports next to where they are used, particularly for uncommonly used features that we want to highlight.
- [ ] All `GIS` object instantiations are one of the following?
    - `gis = GIS()`
    - `gis = GIS('home')`
    - `gis = GIS(profile="your_online_portal")`
    - `gis = GIS('https://pythonapi.playground.esri.com/portal')`
    - `gis = GIS(profile="your_enterprise_portal")`
- [ ] If this notebook requires setup or teardown, did you add the appropriate code to `./misc/setup.py` and/or `./misc/teardown.py`?
- [ ] If this notebook references any portal items that need to be staged on AGOL/Python API playground, did you coordinate with a Python API team member to stage the item the correct way with the api\_data\_owner user?
- [ ] Code refactored & split out across multiple cells, useful comments?
- [ ] Consistent voice/tense/narrative style? Thoroughly checked for typos?
- [ ] All images used like `` instead of ``? All map widgets contain a static image preview? (Call `mapview_inst.take_screenshot()` to do so)
- [ ] All file paths are constructed in an OS-agnostic fashion with `os.path.join()`? (Instead of `r"\foo\bar"`, `os.path.join(os.path.sep, "foo", "bar")`, etc.)
- [ ] **IF YOU WANT THIS SAMPLE TO BE DISPLAYED ON THE DEVELOPERS.ARCGIS.COM WEBSITE**, ping @ DavidJVitale so he can add it to the list for the next deploy

### CODE REVIEW

File: guide/02-api-overview/release-notes.ipynb [hunk 1]
```diff
@@ -1,5 +1,438 @@
 {
```
Comment (rhea-j) [inline guide/02-api-overview/release-notes.ipynb:1]: <p>https://github.com/ArcGIS/geosaurus/issues/6153 - Adds viewshed function</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1026/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Comment (rhea-j) [inline guide/02-api-overview/release-notes.ipynb:1]: <p><a href="https://developers.arcgis.com/python/api-reference/arcgis.raster.toc.html#raster" target="_blank">Raster</a></p><ul><li>create Raster objects from stac catalogs with&nbsp;<code><a href="https://developers.arcgis.com/python/api-reference/arcgis.raster.toc.html#arcgis.raster.Raster.from_stac_item" target="_blank">from_stac_item(</a>)</code></li></ul><p>Can it be changed to </p><ul><li>create Raster objects from <strong>stac items</strong> with <a href="https://developers.arcgis.com/python/api-reference/arcgis.raster.toc.html#arcgis.raster.Raster.from_stac_item" target="_blank">from_stac_item(</a>)</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1026/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (jyaistMap) [reply guide/02-api-overview/release-notes.ipynb:1]: Done!

Comment (jyaistMap) [reply guide/02-api-overview/release-notes.ipynb:1]: Done!

File: guide/02-api-overview/release-notes.ipynb [hunk 2]
```diff
@@ -1,5 +1,607 @@
...
```
Comment (AtmaMani) [inline guide/02-api-overview/release-notes.ipynb:1]: <p>Can we call out the updates to API ref here. We should be able to merge for Widgets and Features modules</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1026/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (AtmaMani) [inline guide/02-api-overview/release-notes.ipynb:1]: <p>Can we replace the hyperlink for STAC with this: https://stacspec.org/ (which is more high level and public facing than the previous)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1026/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Comment (jyaistMap) [reply guide/02-api-overview/release-notes.ipynb:1]: Yep - added a note about adding text emphasis, links to returned objects, and code snippets to `arcgis.geometry`, `arcgis.features`, and `arcgis.widgets`

Comment (jyaistMap) [reply guide/02-api-overview/release-notes.ipynb:1]: Done!

**Review by AtmaMani — APPROVED** [review]

**Comment by AtmaMani** [comment]
@jyaistMap thank you for compiling these detailed release notes

**Pull request merged**